core/include/detray/navigation/intersection/helix_line_intersector.hpp

```diff
@@ -105,19 +106,15 @@ struct helix_intersector_impl<line2D<algebra_t>, algebra_t> {
         // Path length to the point of closest approach on the track
         // @NOTE Ray intersection algorithm is used for the initial guess on the
         // path length
```

Comment:
Same here. I prefer the original code to stay which is more understandable

Reply:
You really want me to duplicate everything again?! How is that easier to understand?

Can I do that in 1,5 weeks, please, when I have actually time for this?

Reply:
I also have a peer-reviewed paper under review process where the test is based on Newton method and deadline is soon. 1,5 weeks is too far away for me

Reply:
And you have not made a tag yet? Can I make a tag for you and we merge this afterwards or do you need to make more changes still?

Reply:
I need to make more changes. yes. And I don't want to change the explanation from Newton to Newton+bisection for no good reason.

Reply:
Great, I have written the algorithm up like this now and I cannot really go back to a method that is conceptually less safe.

What changes do you need to make in particular? And when is you deadline?

Reply:
I can't wait much longer, so I fully duplicated the code now. Can you tell me if this works for you?